RUN_PROTOCOL.md

```
@@ -0,0 +1,117 @@
+Before any submission run, confirm the CL index is populated and files are on disk:
+
+```powershell
+# Count individual CL files on disk
+(Get-ChildItem "cover-letters\" -File | Where-Object { $_.Name -ne "bulk-export-2026-04-29.md" } | Measure-Object).Count
+
+# Count entries in the index
+node -e "import('js-yaml').then(m => { const f = require('fs'); const idx = m.default.load(f.readFileSync('cover-letters/index.yml','utf8')); console.log('index entries:', idx.templates.length); })"
+```
+
+Both counts should be non-zero and roughly equal. If the index is missing or has 0 entries:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)